### ISSUE TITLE
1.7.1 is broken

### ISSUE THREAD

**Issue by Kikobeats** [PeculiarVentures/webcrypto-core#50]
Hello,

Since the [1.7.1. commit](https://github.com/PeculiarVentures/webcrypto-core/commit/47e57b0c87c55e051102593c0975370520ce6883) the library is not more usable.

It's conflict with other typescript types

```
$ tsc -d -p tsconfig.json
../../node_modules/@types/web/index.d.ts(7,1): error TS6200: Definitions of the following identifiers conflict with those in another file: NodeFilter, XPathNSResolver, SVGMatrix, WebKitCSSMatrix, SVGPoint, SVGRect, location, webkitURL, ImportExportKind, TableKind, ValueType, ExportValue, Exports, ImportValue, Imports, ModuleImports, ElementTagNameMap, name, AlgorithmIdentifier, BigInteger, BinaryData, BlobPart, BodyInit, BufferSource, COSEAlgorithmIdentifier, CSSNumberish, CanvasImageSource, ClipboardItemData, ClipboardItemDataType, ClipboardItems, ConstrainBoolean, ConstrainDOMString, ConstrainDouble, ConstrainULong, DOMHighResTimeStamp, EventListenerOrEventListenerObject, Float32List, FormDataEntryValue, GLbitfield, GLboolean, GLclampf, GLenum, GLfloat, GLint, GLint64, GLintptr, GLsizei, GLsizeiptr, GLuint, GLuint64, HTMLOrSVGImageElement, HTMLOrSVGScriptElement, HashAlgorithmIdentifier, HeadersInit, IDBValidKey, ImageBitmapSource, InsertPosition, Int32List, LineAndPositionSetting, MediaProvider, MessageEventSource, MutationRecordType, NamedCurve, OnBeforeUnloadEventHandler, OnErrorEventHandler, PerformanceEntryList, ReadableStreamController, ReadableStreamDefaultReadResult, ReadableStreamReader, RenderingContext, RequestInfo, TexImageSource, TimerHandler, Transferable, Uint32List, UvmEntries, UvmEntry, VibratePattern, WindowProxy, XMLHttpRequestBodyInit, AlignSetting, AnimationPlayState, AnimationReplaceState, AppendMode, AttestationConveyancePreference, AudioContextLatencyCategory, AudioContextState, AuthenticatorAttachment, AuthenticatorTransport, AutoKeyword, AutomationRate, BinaryType, BiquadFilterType, CanPlayTypeResult, CanvasDirection, CanvasFillRule, CanvasFontKerning, CanvasFontStretch, CanvasFontVariantCaps, CanvasLineCap, CanvasLineJoin, CanvasTextAlign, CanvasTextBaseline, CanvasTextRendering, ChannelCountMode, ChannelInterpretation, ClientTypes, ColorGamut, ColorSpaceConversion, CompositeOperation, CompositeOperationOrAuto, ConnectionType, CredentialMediationRequirement, DOMParserSupportedType, DirectionSetting, DisplayCaptureSurfaceType, DistanceModelType, DocumentReadyState, EndOfStreamError, EndingType, FillMode, FontFaceLoadStatus, FontFaceSetLoadStatus, FullscreenNavigationUI, GamepadHapticActuatorType, GamepadMappingType, HdrMetadataType, IDBCursorDirection, IDBRequestReadyState, IDBTransactionMode, ImageOrientation, ImageSmoothingQuality, IterationCompositeOperation, KeyFormat, KeyType, KeyUsage, LineAlignSetting, MediaDecodingType, MediaDeviceKind, MediaEncodingType, MediaKeyMessageType, MediaKeySessionClosedReason, MediaKeySessionType, MediaKeyStatus, MediaKeysRequirement, MediaSessionAction, MediaSessionPlaybackState, MediaStreamTrackState, NavigationType, NotificationDirection, NotificationPermission, OrientationLockType, OrientationType, OscillatorType, OverSampleType, PanningModelType, PaymentComplete, PermissionName, PermissionState, PlaybackDirection, PositionAlignSetting, PredefinedColorSpace, PremultiplyAlpha, PresentationStyle, PublicKeyCredentialType, PushEncryptionKeyName, RTCBundlePolicy, RTCDataChannelState, RTCDegradationPreference, RTCDtlsTransportState, RTCIceCandidateType, RTCIceComponent, RTCIceConnectionState, RTCIceCredentialType, RTCIceGathererState, RTCIceGatheringState, RTCIceProtocol, RTCIceTcpCandidateType, RTCIceTransportPolicy, RTCIceTransportState, RTCPeerConnectionState, RTCPriorityType, RTCRtcpMuxPolicy, RTCRtpTransceiverDirection, RTCSdpType, RTCSignalingState, RTCStatsIceCandidatePairState, RTCStatsType, ReadyState, RecordingState, ReferrerPolicy, RemotePlaybackState, RequestCache, RequestCredentials, RequestDestination, RequestMode, RequestRedirect, ResidentKeyRequirement, ResizeObserverBoxOptions, ResizeQuality, ResponseType, ScrollBehavior, ScrollLogicalPosition, ScrollRestoration, ScrollSetting, SecurityPolicyViolationEventDisposition, SelectionMode, ServiceWorkerState, ServiceWorkerUpdateViaCache, ShadowRootMode, SlotAssignmentMode, SpeechSynthesisErrorCode, TextTrackKind, TextTrackMode, TouchType, TransferFunction, UserVerificationRequirement, VideoFacingModeEnum, WebGLPowerPreference, WorkerType, XMLHttpRequestResponseType
../../node_modules/@types/web/index.d.ts(221,5): error TS2374: Duplicate index signature for type 'string'.
../../node_modules/@types/web/index.d.ts(269,5): error TS2687: All declarations of 'privateKey' must have identical modifiers.
../../node_modules/@types/web/index.d.ts(270,5): error TS2687: All declarations of 'publicKey' must have identical modifiers.
../../node_modules/@types/web/index.d.ts(675,5): error TS2374: Duplicate index signature for type 'string'.
../../node_modules/@types/web/index.d.ts(1142,5): error TS2374: Duplicate index signature for type 'string'.
../../node_modules/@types/web/index.d.ts(1697,5): error TS2717: Subsequent property declarations must have the same type.  Property 'transfer' must be of type 'any[]', but here has type 'Transferable[]'.
../../node_modules/@types/web/index.d.ts(2691,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(3235,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(3347,5): error TS2717: Subsequent property declarations must have the same type.  Property 'globalCompositeOperation' must be of type 'string', but here has type 'GlobalCompositeOperation'.
../../node_modules/@types/web/index.d.ts(4013,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(4047,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(4057,5): error TS2374: Duplicate index signature for type 'string'.
../../node_modules/@types/web/index.d.ts(4123,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(4199,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(5144,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(5902,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(6157,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(6173,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(6479,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(6480,5): error TS2374: Duplicate index signature for type 'string'.
../../node_modules/@types/web/index.d.ts(6820,5): error TS2717: Subsequent property declarations must have the same type.  Property 'loading' must be of type 'string', but here has type '"eager" | "lazy"'.
../../node_modules/@types/web/index.d.ts(7772,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(9273,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(9622,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(9749,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(10023,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(10039,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(10699,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(10721,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(11411,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'Request' must be of type '{ new (input: RequestInfo, init?: RequestInit): Request; prototype: Request; }', but here has type '{ new (input: RequestInfo | URL, init?: RequestInit): Request; prototype: Request; }'.
../../node_modules/@types/web/index.d.ts(12514,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(12654,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(12705,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(12916,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(13133,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(13482,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(13504,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(13515,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(13668,5): error TS2374: Duplicate index signature for type 'string'.
../../node_modules/@types/web/index.d.ts(13735,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(13990,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(14014,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(14086,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/@types/web/index.d.ts(16578,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(25,1): error TS6200: Definitions of the following identifiers conflict with those in another file: NodeFilter, XPathNSResolver, SVGMatrix, WebKitCSSMatrix, SVGPoint, SVGRect, location, webkitURL, ImportExportKind, TableKind, ValueType, ExportValue, Exports, ImportValue, Imports, ModuleImports, ElementTagNameMap, name, AlgorithmIdentifier, BigInteger, BinaryData, BlobPart, BodyInit, BufferSource, COSEAlgorithmIdentifier, CSSNumberish, CanvasImageSource, ClipboardItemData, ClipboardItemDataType, ClipboardItems, ConstrainBoolean, ConstrainDOMString, ConstrainDouble, ConstrainULong, DOMHighResTimeStamp, EventListenerOrEventListenerObject, Float32List, FormDataEntryValue, GLbitfield, GLboolean, GLclampf, GLenum, GLfloat, GLint, GLint64, GLintptr, GLsizei, GLsizeiptr, GLuint, GLuint64, HTMLOrSVGImageElement, HTMLOrSVGScriptElement, HashAlgorithmIdentifier, HeadersInit, IDBValidKey, ImageBitmapSource, InsertPosition, Int32List, LineAndPositionSetting, MediaProvider, MessageEventSource, MutationRecordType, NamedCurve, OnBeforeUnloadEventHandler, OnErrorEventHandler, PerformanceEntryList, ReadableStreamController, ReadableStreamDefaultReadResult, ReadableStreamReader, RenderingContext, RequestInfo, TexImageSource, TimerHandler, Transferable, Uint32List, UvmEntries, UvmEntry, VibratePattern, WindowProxy, XMLHttpRequestBodyInit, AlignSetting, AnimationPlayState, AnimationReplaceState, AppendMode, AttestationConveyancePreference, AudioContextLatencyCategory, AudioContextState, AuthenticatorAttachment, AuthenticatorTransport, AutoKeyword, AutomationRate, BinaryType, BiquadFilterType, CanPlayTypeResult, CanvasDirection, CanvasFillRule, CanvasFontKerning, CanvasFontStretch, CanvasFontVariantCaps, CanvasLineCap, CanvasLineJoin, CanvasTextAlign, CanvasTextBaseline, CanvasTextRendering, ChannelCountMode, ChannelInterpretation, ClientTypes, ColorGamut, ColorSpaceConversion, CompositeOperation, CompositeOperationOrAuto, ConnectionType, CredentialMediationRequirement, DOMParserSupportedType, DirectionSetting, DisplayCaptureSurfaceType, DistanceModelType, DocumentReadyState, EndOfStreamError, EndingType, FillMode, FontFaceLoadStatus, FontFaceSetLoadStatus, FullscreenNavigationUI, GamepadHapticActuatorType, GamepadMappingType, HdrMetadataType, IDBCursorDirection, IDBRequestReadyState, IDBTransactionMode, ImageOrientation, ImageSmoothingQuality, IterationCompositeOperation, KeyFormat, KeyType, KeyUsage, LineAlignSetting, MediaDecodingType, MediaDeviceKind, MediaEncodingType, MediaKeyMessageType, MediaKeySessionClosedReason, MediaKeySessionType, MediaKeyStatus, MediaKeysRequirement, MediaSessionAction, MediaSessionPlaybackState, MediaStreamTrackState, NavigationType, NotificationDirection, NotificationPermission, OrientationLockType, OrientationType, OscillatorType, OverSampleType, PanningModelType, PaymentComplete, PermissionName, PermissionState, PlaybackDirection, PositionAlignSetting, PredefinedColorSpace, PremultiplyAlpha, PresentationStyle, PublicKeyCredentialType, PushEncryptionKeyName, RTCBundlePolicy, RTCDataChannelState, RTCDegradationPreference, RTCDtlsTransportState, RTCIceCandidateType, RTCIceComponent, RTCIceConnectionState, RTCIceCredentialType, RTCIceGathererState, RTCIceGatheringState, RTCIceProtocol, RTCIceTcpCandidateType, RTCIceTransportPolicy, RTCIceTransportState, RTCPeerConnectionState, RTCPriorityType, RTCRtcpMuxPolicy, RTCRtpTransceiverDirection, RTCSdpType, RTCSignalingState, RTCStatsIceCandidatePairState, RTCStatsType, ReadyState, RecordingState, ReferrerPolicy, RemotePlaybackState, RequestCache, RequestCredentials, RequestDestination, RequestMode, RequestRedirect, ResidentKeyRequirement, ResizeObserverBoxOptions, ResizeQuality, ResponseType, ScrollBehavior, ScrollLogicalPosition, ScrollRestoration, ScrollSetting, SecurityPolicyViolationEventDisposition, SelectionMode, ServiceWorkerState, ServiceWorkerUpdateViaCache, ShadowRootMode, SlotAssignmentMode, SpeechSynthesisErrorCode, TextTrackKind, TextTrackMode, TouchType, TransferFunction, UserVerificationRequirement, VideoFacingModeEnum, WebGLPowerPreference, WorkerType, XMLHttpRequestResponseType
../../node_modules/typescript/lib/lib.dom.d.ts(239,5): error TS2374: Duplicate index signature for type 'string'.
../../node_modules/typescript/lib/lib.dom.d.ts(287,5): error TS2687: All declarations of 'privateKey' must have identical modifiers.
../../node_modules/typescript/lib/lib.dom.d.ts(288,5): error TS2687: All declarations of 'publicKey' must have identical modifiers.
../../node_modules/typescript/lib/lib.dom.d.ts(681,5): error TS2374: Duplicate index signature for type 'string'.
../../node_modules/typescript/lib/lib.dom.d.ts(1125,5): error TS2374: Duplicate index signature for type 'string'.
../../node_modules/typescript/lib/lib.dom.d.ts(2625,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(3158,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(3919,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(3953,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(3963,5): error TS2374: Duplicate index signature for type 'string'.
../../node_modules/typescript/lib/lib.dom.d.ts(4029,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(4105,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(5023,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(5741,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(5996,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(6012,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(6303,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(6304,5): error TS2374: Duplicate index signature for type 'string'.
../../node_modules/typescript/lib/lib.dom.d.ts(7594,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(9058,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(9407,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(9534,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(9795,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(9811,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(10469,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(10491,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(12198,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(12338,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(12389,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(12600,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(12817,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(13156,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(13178,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(13189,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(13342,5): error TS2374: Duplicate index signature for type 'string'.
../../node_modules/typescript/lib/lib.dom.d.ts(13408,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(13663,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(13687,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(13759,5): error TS2374: Duplicate index signature for type 'number'.
../../node_modules/typescript/lib/lib.dom.d.ts(16230,5): error TS2374: Duplicate index signature for type 'number'.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

If I lock the version to 1.7.0 it works fine, although ideally I want ot use the latest version available

**Comment by microshine** [comment]
I've published `v1.7.2` please try it. I removed `@types/web` lib from dependencies

**Comment by Kikobeats** [comment]
Hello, I confirm `v1.7.2` is working fine at my end, thanks a lot 🙂
